force-app/main/default/classes/MaintenanceRequestHelper.cls

```diff
@@ -0,0 +1,57 @@
+public with sharing class MaintenanceRequestHelper {
+
+    public static void createMaintenanceRequestRecord(List<Case> caseList) {
+        try{
+            List<Case> newCases = new List<Case>();
+            Integer cycle = getMaintenanceCycle(caseList) == null ? 0 : getMaintenanceCycle(caseList);
+            for(Case c: caseList) {
+                Case newCase = new Case();
+                newCase.Vehicle__c = c.Vehicle__c;
+                newCase.Status = 'New';
+                newCase.Origin ='Web';
+                newCase.Type = 'Routine Maintenance';
+                newCase.Subject = 'Clone of ' + c.Subject ;
+                newCase.Date_Reported__c = Date.today();
+                newCase.OldId__c = c.Id;
+                newCase.Date_Due__c = Date.today().addDays(cycle);
+                newCases.add(newCase);
+            }
+            if(newCases.size() > 0) {
+                insert newCases;
+                createEquipmentMaintenanceItems(newCases);
+            }
+        }catch(Exception e) {
```

Comment:
what exception type are you expecting to catch here? why are you using try/catch at all? if you are expecting any errors might happen then try to implement it the way that error will never happen

Reply:
remind me during the call regarding this place please